python-web-applications/requirements.txt

```
@@ -0,0 +1 @@
+flask
```

👩‍💻 **Technical Recommendation**

We should be instilling good programming practices in our readers, so let's specify a Flask version. That will mitigate potential problems with breaking changes in Flask when someone reads this article in the future. We've had a similar situation in a Django article before.

```suggestion
Flask==1.1.2
```